changes/ce/fix-13164.en.md

```diff
@@ -0,0 +1,6 @@
+Fix HTTP authorization request body encoding.
+
+Prior to this fix, the HTTP authorization request body encoding format was taken from the `accept` header.
+The fix is to respect the `content-type` header instead.
+Also added `access` templating variable for v4 compatibility.
+The access code of SUBSCRIBE action is `1` and SUBSCRIBE action is `2`.
```

Comment:
Is there a typo in line 6? : The access code of SUBSCRIBE action is `1` and SUBSCRIBE action is `2`.

Reply:
PUBLISH is 2.
will fix in the next PR.
Thanks.

Reply:
fixed.